test/converter-test.js

```diff
@@ -190,6 +192,19 @@ describe('Converter', () => {
       });
     });
 
+    context('inches', () => {
```

Comment:
There is already an "inches" context (line 167 at current time of comment), can you merge this test into the existing context?

Reply:
Sure, will do this @cannawen.
Can you point me to some documentation regarding how `conversion_helper-test.js` functions, I'm not very sure I understand it.

Reply:
There's no documentation, sorry XD 

It's a file where each of the functions in conversion_helper is tested individually. So you could add a `describe('#preprocessComment()', () => { }` block and add a test inside that solely tests the operations `preprocessComment` does (a unit test). 

So your test might look something like:
```
      it('should convert fractions into decimals', () => {
        const comment = createComment("subredditname", "post title", "post text 9/10 miles");
        ch.preprocessComment(comment)['body'].should.equal("post text 0.9 miles");
      });
```

I hope that made sense, let me know if any part of that was unclear!

Reply:
@cannawen did those changes, but discovered an issue, turns out `numberRegex` escapes as soon as possible, i.e. for `3,210` it only captures `3` and not the whole number. Shouldn't this be fixed to work lazily instead?

Reply:
Oops, realised why you were using `endregex` and `startregex` everywhere. 😛 Will fix this, sorry.

Reply:
I'm not super familiar with lazy regexes, but if you think that will fix a bug, go for it! `,` is also part of the `endRegex` as well so I don't know if that will cause an issue for you

Reply:
I don't think it's very easy to do fix the regex either. For now, I've made my own regex to detect ending. I think I've fixed the rest, give it a review @cannawen.